configure.ac

```diff
@@ -10,7 +10,7 @@
 AC_PREREQ(2.52)
 AC_INIT([scribe], [1.5.0])
 AC_CONFIG_MACRO_DIR([aclocal])
-AM_INIT_AUTOMAKE([foreign -Wall])
+#AM_INIT_AUTOMAKE([foreign -Wall])
```

Comment:
I faced the same issue, and also did the same fix as done in this pull request https://github.com/facebookarchive/scribe/pull/83

From the conversation "The bug is due to Scribe calling AM_INIT_AUTOMAKE in both the top of configure.ac and inside FB_INITIALIZE defined in acinclude.m4. The solution is to comment the AM_INIT_AUTOMAKE"